data/templates/esx-ks

```diff
@@ -209,7 +209,20 @@ cp /var/log/esxi_install.log "/vmfs/volumes/datastore1/firstboot-esxi_install.lo
 #
 # NOTE: this script will execute right away as a result of writing it to local.sh
 # along with executing on every subsequent boot
-wget http://<%=server%>:<%=port%>/api/current/templates/<%=rackhdCallbackScript%>?nodeId=<%=nodeId%> -O /etc/rc.local.d/local.sh
+#
+# Try to download call back script 60 times 1 second
+# sleep in between to allow link to be up after DHCP
+for retry in $(awk 'BEGIN { for ( i=0; i<60; i++ ) { print i; } }');
```

Comment:
This iteration technique seems overly complex.  How about:

for retry in $(seq 1 60); do
    ...
    ...
done